apps/docs/content/components/input-otp/allowedKeys.ts

```diff
@@ -0,0 +1,27 @@
+const App = `import {InputOtp} from "@nextui-org/react";
+
+export default function App() {
+  const exps = [
+    {name: "For below InputOtp, only small alphabets (a to z) are allowed:", value: "^[a-z]*$"} , 
+    {name: "For below InputOtp, only big alphabets(A to Z) are allowed:", value: "^[A-Z]*$"}
```

Comment:
big alphabets -> upper case alphabets

Reply:
done in: cab6a940e04a8e4d75fd89529b1dafb6bbc33a5c